packages/at_persistence_secondary_server/lib/src/model/at_metadata_builder.dart

```diff
@@ -121,6 +76,8 @@ class AtMetadataBuilder {
       atMetaData.expiresAt = _getExpiresAt(
           currentUtcTimeToMillisecondPrecision.millisecondsSinceEpoch, ttl,
           ttb: ttb);
+    } else {
+      atMetaData.expiresAt = null;
```

Comment:
if ttl is null then expiresAt should be null

Reply:
Worth checking for scenarios where:

- expiresAt has been set and setTTL is always called, even when the TTL value is null.

Same for ttb/availableAt and ttr/refreshAt

Reply:
Thanks @XavierChanth ... I believe from memory that those fields are only ever set here but I will double check